salt/cloud/clouds/vmware.py

```diff
@@ -776,23 +820,22 @@ def _wait_for_ip(vm_ref, max_wait):
     vmware_tools_status = _wait_for_vmware_tools(vm_ref, max_wait_vmware_tools)
     if not vmware_tools_status:
         return False
-
     time_counter = 0
     starttime = time.time()
     while time_counter < max_wait_ip:
         if time_counter % 5 == 0:
             log.info("[ {0} ] Waiting to retrieve IPv4 information [{1} s]".format(vm_ref.name, time_counter))
 
-        if vm_ref.summary.guest.ipAddress and match(IP_RE, vm_ref.summary.guest.ipAddress) and vm_ref.summary.guest.ipAddress != '127.0.0.1':
-            log.info("[ {0} ] Successfully retrieved IPv4 information in {1} seconds".format(vm_ref.name, time_counter))
+	if vm_ref.summary.guest.ipAddress and valid_ip(vm_ref.summary.guest.ipAddress):
```

Comment:
I think the tabs are mixed with spaces here. Can you fix it by using 4 spaces for all the tabs?